x-pack/legacy/plugins/siem/server/lib/detection_engine/scripts/signals/query_signals.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+#
+# Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+# or more contributor license agreements. Licensed under the Elastic License;
+# you may not use this file except in compliance with the Elastic License.
+#
+
+set -e
+./check_env_variables.sh
+
+# Example: ./signals/query_signals.sh
+  curl -v -k \
```

Comment:
Woa, I don't think you want the `-v` here for the example. That adds a lot of extra stuff! :-)

Reply:
very true - was debugging and forgot to take it out. Thanks 👍